alf/algorithms/entropy_target_algorithm.py

```diff
@@ -32,7 +33,18 @@
 
 
 @alf.configurable
-class EntropyTargetAlgorithm(Algorithm):
+def EntropyTargetAlgorithm(action_spec,
```

Comment:
this is no longer an Algorithm, maybe call it entropy_target_algorithm_ctor?

Reply:
> this is no longer an Algorithm, maybe call it entropy_target_algorithm_ctor?

@emailweixu other way of creating a separate NestedEntropyTargetAlgorithm class, keeping the current EntropyTargetAlgorithm unchanged also looks good to me.